napi-inl.h

```diff
@@ -205,14 +205,19 @@ struct FinalizeData {
     });
   }
 
-#ifdef NODE_API_EXPERIMENTAL_HAS_POST_FINALIZER
+#if defined(NODE_API_EXPERIMENTAL_HAS_POST_FINALIZER)
```

Comment:
Is this change needed?

Reply:
Whoops, this was a leftover of a different implementation. Addressed in https://github.com/nodejs/node-addon-api/pull/1568/commits/23b57cc93946df51f30cbfada61213be14f61421